cmd/subsyStems.go

```diff
@@ -209,6 +209,14 @@ func newSubSystem(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	forbiddenNames := []string{"completion", "help", "pkgmanagers", "stacks", "subsystems"}
```

Comment:
I don't think hard coding is a good solution here.
Every dev implementing a new command would have to think about this and add it to this list.

Reply:
I'm thinking of doing:
```bash
for _, command := range cmd.Root().Commands() {
        if subSystemName == command.Name() {
            cmdr.Error.Printfln(apx.Trans(“subsystems.new.error.forbiddenName”), subSystemName)
            return nil
        }
    }
```
and move it after subsystems.new.error.alreadyExists, I don't know where to get only commands(apx completion, apx subsystems) without existing subsystems commands, what do you think?

Reply:
Sounds like a valid approach, yeah.